resources/prefs.json

```diff
@@ -100,7 +100,7 @@
   "layout.columns.enabled": false,
   "layout.flexbox.enabled": false,
   "layout.legacy_layout": false,
-  "layout.tables.enabled": false,
```

Comment:
Whoops. I've disabled this. It was accidentally left on from our experiments. I think it was useful though, because we know more-or-less how the tests looks now.